### PR TITLE
New package: visitors-0.7

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

File: srcpkgs/visitors/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'visitors'
+pkgname=visitors
+version=0.7
+revision=1
+wrksrc="${pkgname}_${version}"
+short_desc="Lightweight web log analyzer"
+maintainer="Adrian Siekierka <asiekierka@gmail.com>"
+license="GPL-2"
+homepage="http://www.hping.org/visitors/"
+distfiles="http://www.hping.org/visitors/visitors-0.7.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/visitors/template]: Use `${version}`

Comment (asiekierka) [inline srcpkgs/visitors/template]: Forgot! Sorry.

File: srcpkgs/visitors/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'visitors'
+pkgname=visitors
+version=0.7
+revision=1
+wrksrc="${pkgname}_${version}"
+short_desc="Lightweight web log analyzer"
+maintainer="Adrian Siekierka <asiekierka@gmail.com>"
+license="GPL-2"
+homepage="http://www.hping.org/${pkgname}/"
```
Comment (Vaelatern) [inline srcpkgs/visitors/template]: No `${pkgname}` in this please :)

File: srcpkgs/visitors/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'visitors'
+pkgname=visitors
+version=0.7
+revision=1
+wrksrc="${pkgname}_${version}"
+short_desc="Lightweight web log analyzer"
+maintainer="Adrian Siekierka <asiekierka@gmail.com>"
+license="GPL-2"
```
Comment (Vaelatern) [inline srcpkgs/visitors/template]: Not adequate in this case. There is custom copyright if you read the COPYING file you also need to account for.

Comment (asiekierka) [inline srcpkgs/visitors/template]: How is it not adequate? The copyright notice is MIT, while the compiled binary (what's being packaged) is GPLv2 - the complete product as a whole is then also GPLv2.

**Comment by Vaelatern** [comment]
You don't want the `iis2apache.pl` file? What about the .png files?

**Comment by asiekierka** [comment]
@Vaelatern We'd have to rename it to something like visitors-iis2apache to prevent potential conflicts with other software? As for the PNG files, I don't particularly see why? They're useful for the HTML mode, but that always uses relative paths so they're not useful in a packaged variant.

File: srcpkgs/visitors/template (hunk 3 above)
Comment (Vaelatern) [inline srcpkgs/visitors/template]: Then `license="GPL-2, MIT"` but the MIT needs a vlicense of it's own, so you may want to chop out the first X lines of the file and put that as a license.

**Comment by Vaelatern** [comment]
Okay, no png files. There is no other `iis2apache` in the repos.

**Comment by Vaelatern** [comment]
Nevermind about the `iis2apache` file, turns out if someone really needs IIS stuff, they know enough to find it.
Currently waiting on you setting accurate license information, and vlicensing the BSD license from the C file.

**Pull request merged**